services/wallet/router.go

```diff
@@ -399,10 +399,14 @@ func (r *Router) suggestedRoutes(
 		return nil, err
 	}
 
-	prices, err := fetchCryptoComparePrices([]string{"ETH", tokenSymbol}, "USD")
+	pricesMap, err := fetchCryptoComparePrices([]string{"ETH", tokenSymbol}, []string{"USD"})
```

Comment:
@dlipicar don't we need to calculate a suggested root in the user's selected currency?

Reply:
Hmm. I didn't really change anything here, just changed the string argument "USD" to a vector containing that single argument, because the method signature changed.
What you say makes sense to me (although I'm guessing the route that minimizes fees will be the same for any currency? IDK) This might be a question for @Khushboo-dev-cpp or @alaibe

Reply:
@dlipicar I know that approach was already here, but since I see it for the first time I just wanted to clarify that.

Reply:
Discussed with Anthony and Khushbool, this will be changed to use the user's currency